src/client/components/NavigationBar/style.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../style';
+@import url('https://fonts.googleapis.com/css?family=Lato');
```

Comment:
- we should use the Lato bold version here
- you already import the style.scss what import the lato, so if you want to use the Lato simple you dont need to import any font here